sphinx/writers/latex.py

```diff
@@ -505,8 +507,7 @@ class LaTeXTranslator(nodes.NodeVisitor):
 
     def __init__(self, document, builder):
         # type: (nodes.document, LaTeXBuilder) -> None
-        super(LaTeXTranslator, self).__init__(document)
-        self.builder = builder
+        super(LaTeXTranslator, self).__init__(builder, document)
```

Comment:
Unfortunately, the order of arguments are different between Translators.
I think it's time to integrate to one way.

Reply:
It must be a breaking change. So I'll post this as another PR.